package.json

```diff
@@ -66,7 +66,7 @@
     "cssnano": "^4.1.10",
     "debug": "^4.1.1",
     "es-module-lexer": "^0.3.18",
-    "esbuild": "^0.2.6",
+    "esbuild": "^0.3.2",
```

Comment:
Should commit `yarn.lock` with this.

Reply:
I noticed. I was using npm, installing yarn right now.